### DISCUSSION TITLE
VDR gene as control region

### DISCUSSION THREAD

**Discussion by emeadd** [pypgx/pypgx#91] [Q&A] [answered]
Hello.  I am also interested in CYP2D6 genotyping/phenotyping using WES data.  Why do you use the VDR gene to compute control statistics?  Thank you for your help.

**Comment by sbslee** [comment, marked as answer]
@emeadd,

That's a great question!

First of all, you don't have to use the VDR gene for the `compute-control-statistics` command. If you look at the command's [docs](https://pypgx.readthedocs.io/en/latest/cli.html#compute-control-statistics), you will see that it actually accepts two other genes (EGFR and RYR1) or a custom region of your choice.

```
$ pypgx compute-control-statistics \
EGFR \
control-statistics.zip \
1.bam 2.bam

$ pypgx compute-control-statistics \
chr1:100-200 \
control-statistics.zip \
1.bam 2.bam
```

So why were the three genes chosen by PyPGx in the first place? From one of my Stargazer papers (see the [Citation](https://pypgx.readthedocs.io/en/latest/readme.html#citation) page for more details):

> In the initial development of Stargazer, three genes—VDR, RYR1, and EGFR—were evaluated as control loci. ... They are also reported to exhibit low rates of whole gene deletion and/or duplication according to the Database of Genomic Variants. All three genes produced the same copy number results for CYP2D6 and CYP2D7.

So the bottom line is, as long as you are providing a region that's known to be stably maintained as two copies AND is fairly large, you can pick and choose whichever gene you'd like.

Let me know if you have further questions.

**Reply by emeadd** [reply]
Thank you!
